shaikram_abdulguro/activity1/index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css">
+    <title>Log In</title>
+</head>
+<style>
+    body{
+        background: #2d2f30;
+    }
+    h1{
+        color: blue;
+    }
+    h3{
+        color: gray;
+    }
+    form{
+        margin-top: 5em;
+        width:500px;
+        height:auto;
+        border: 1px solid grey;
+        background:#fff;
+        padding: 2em;
+        border-radius: 3px;
+    }
+    h3, span, label{
+        float: left;
+    }
+</style>
+<body>
+    <center>
+        
+        <div class="container ">
+            <form onsubmit="validate();">
+                <div class="mb-3">
+                    <qh1>Log-In Page</qh1>
+                </div>
+                <div class="mb-3">
+                    <h3>Login using given data below:</h3>
+                    <span>Email: <b>kodego@yahoo.com</b></span><br><br><br>
+                    <span>Password: <b>webdevelopment</b></span><br><br>
+                   
+                </div>
+                <div class="mb-3">
+                  <label for="exampleInputEmail1" class="form-label">Email address</label>
+                  <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" minlength="8" maxlength="25" required>
+                </div>
+                <div class="mb-3">
+                  <label for="exampleInputPassword1" class="form-label">Password</label>
+                  <input type="password" class="form-control" id="exampleInputPassword1" minlength="8" maxlength="25" required>
+                </div>
+                
+                <button type="submit" class="btn btn-primary">Submit</button>
+              </form>
+        </div>
+    </center>
+    
+</body>
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/js/bootstrap.min.js" integrity="sha384-ODmDIVzN+pFdexxHEHFBQH3/9/vQ9uori45z4JjnFsRydbmQbmL5t1tQ0culUzyK" crossorigin="anonymous"></script>
+<script>
+    function validate(){
+        var email = document.getElementById('exampleInputEmail1');
+        var password = document.getElementById('exampleInputPassword1');
+        var loginData = ["kodego@yahoo.com", "webdevelopment"];
+
+        if(email.value == loginData[0] && password.value == loginData[1]){
```

Comment:
spacing , formatting
if ( condition && condition ) {

}

also if you are conditioning a strict values use ===